src/components/PersonDetails/index.tsx

```diff
@@ -15,8 +15,8 @@ import { groupBy } from 'lodash';
 import PageTitle from '../Common/PageTitle';
 
 const messages = defineMessages({
-  appearsin: 'Appears in',
-  crewmember: 'Crew Member',
+  appearsin: 'Acting',
```

Comment:
Not everything under here is "acting." There are talk shows, documentaries, etc.